microreactors/gcmr/assembly/RIA_steady_state/hpc_tests

```diff
@@ -0,0 +1,38 @@
+[Tests]
+  # Multiphysics
+  [steady_multiphysics]
+    type = 'CSVDiff'
+    input = 'Griffin_steady_state.i'
+    csvdiff = 'gcmr_steady_out.csv'
+    executable_pattern = 'blue_crab*'
+    cli_args = "Outputs/file_base=gcmr_steady_out
+                Executioner/set_sam_default_petsc_options=false"
+    skip = '#506'
+    # Need enough memory
+    min_parallel = 196
+    max_time = 3000
+    # Griffin overrode the material property output and is outputting warnings in BlueCrab
+    allow_warnings = true
+    allow_unused = true
+  []
+
+  # Multiphysics
+  [transient_multiphysics]
+    type = 'CSVDiff'
+    working_directory = '../RIA_transient'
```

Comment:
This won't work with the output options

```
[0.000s]     SKIP microreactors/gcmr/assembly/steady_state.transient_multiphysics [--sep-files* enabled]
```

Reply:
isnt sep-files just for the console output that gets captured in a text file?

does it also change the directory output of checkpoints / exodus files/ etc, that are not in the list of gold files?